vassal-app/src/main/java/VASSAL/counters/FreeRotator.java

```diff
@@ -202,6 +215,10 @@ public double getCumulativeAngleInRadians() {
     return -PI_180 * getCumulativeAngle();
   }
 
+  public boolean isFreeRotation() {
+    return validAngles.length == 1;
+  }
+
   public void setAngle(double angle) {
     if (validAngles.length == 1) {
```

Comment:
Use `isFreeRotation()` here?

Reply:
Yes indeed, and the other dozen or so places. Fixed.